adminSiteClient/admin.scss

```diff
@@ -1224,4 +1224,27 @@ main:not(.ChartEditorPage):not(.GdocsEditPage) {
         color: #007bff;
         margin-bottom: 8px;
     }
+
+    .ImageIndexPage__alt-text-editor {
+        textarea {
+            border-radius: 6px;
+            position: relative;
+            border: 1px solid #ccc;
+            width: 100%;
+            &.ImageIndexPage__alt-text-editor--should-autosize {
+                field-sizing: content;
```

Comment:
This is only supported in chrome as of now, but I think that's the browser most of the authors use.